src/app/public/modules/sectioned-form/fixtures/sectioned-form-no-sections.component.fixture.html

```diff
@@ -1,2 +1,3 @@
-<sky-sectioned-form>
-</sky-sectioned-form>
\ No newline at end of file
+<sky-sectioned-form
+  [maintainTabContent]="maintainTabContent">
```

Comment:
Closing bracket needs to be on a new line.

Reply:
Done